src/Misc/layoutroot/run.sh

```diff
@@ -53,6 +53,34 @@ runWithManualTrap() {
     done
 }
 
+# update ca certificates in case they are injected with a volume mount
```

Comment:
update the comment to be just 
```
update ca certificates via `update-ca-certificates` or `update-ca-trust`
```
?

Reply:
Maybe I should delete the comment?
I don't want to tie it up with implementation in case we end up using another tool if needed, and the name should be enough... Does not have to be a volume mount
What do you think?

Reply:
sure, removing also seems fine to me. 😄